architecture/additional_concepts/storage.adoc

```diff
@@ -254,6 +254,23 @@ take advantage of the volume's ability for ROX. Errors in the provider show up
 at runtime as mount errors.
 ====
 
+The table below lists the access modes supported by different persistent volumes.
+
+.Supported Access Modes for Persistent Volumes
+[cols=",^v,^v,^v", width="100%",options="header"]
+|===
+|Volume Plugin  |ReadWriteOnce  |ReadOnlyMany  |ReadWriteMany  
+|NFS  | X | X | X 
+|HostPath  | X | - |  -
+|GlusterFS  | X | X |  X
+|Ceph RBD  | X | X |  X
+|Openstack Cinder  | X | - |  -
+|AWS EBS  | X | - |  -
+|GCE Persistent Disk  | X | X |  -
+|iSCSI  | X | X |  -
+|Fiber Channel  | X | X |  -
+|===
```

Comment:
@rootfs I only included the ones supported by OpenShift, as listed in https://docs.openshift.com/container-platform/3.3/architecture/additional_concepts/storage.html#types-of-persistent-volumes 

Do we need to include the suggested ones as well? OR maybe we can change the description to say "Access modes for persistent volumes supported by OpenShift"